test/t8n/t8n.cpp

```diff
@@ -103,11 +103,24 @@ int main(int argc, const char* argv[])
                     tx.chain_id = chain_id;
 
                     auto res = state::transition(state, block, tx, rev, vm);
+
+                    const auto tx_computed_hash = keccak256(rlp::encode(tx));
```

Comment:
```suggestion
                    const auto computed_tx_hash = keccak256(rlp::encode(tx));
```
Keep "tx hash" words together.